### PR TITLE
Hotfix/fix version warnings

### LINKED ISSUE #27: Path to composer.lock
I know this SDK is still beta only, but I'm using it as a base for our integration.

One error that occurred to me is that in `\SwedbankPay\Api\Client\Version` you're trying to locate composer.lock using current working directory (`getcwd()`). In our case that won't work because that is another folder. 

For us, using the Laravel helper `base_path('composer.lock')` works much better.

Just wanted to leave a note here if someone else having problems with this.

### PR DESCRIPTION
**Pull request by aait**
Fixes #27.

### CODE REVIEW

File: src/SwedbankPay/Api/Client/ClientVersion.php [hunk 1]
```diff
@@ -143,6 +143,9 @@ private function tryGetVersionNumberFromComposerJson(&$version) : bool
     private function tryGetVersionNumberFromComposerLock(&$version) : bool
     {
         $composerLockPath = getcwd() . DIRECTORY_SEPARATOR . 'composer.lock';
+        if (!file_exists($composerLockPath)) {
```
Comment (payex-dev) [inline src/SwedbankPay/Api/Client/ClientVersion.php:146]: Codacy Issue found: [The use of function file_exists() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5523738)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/SwedbankPay/Api/Client/ClientVersion.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5523738&bid=18098346)

**Comment by aait** [comment]
> Why are `Client.php` and `Response.php` deleted?

They were deprecated and were duplicated.
See replacement 
https://github.com/SwedbankPay/swedbank-pay-sdk-php/blob/master/src/SwedbankPay/Api/Client/Client.php
https://github.com/SwedbankPay/swedbank-pay-sdk-php/blob/master/src/SwedbankPay/Api/Service/Response.php

Depecated classes were used by woocommerce plugin. It don't use it now, so it can be removed.

**Comment by aait** [comment]
This PR is related ro https://github.com/SwedbankPay/swedbank-pay-sdk-php/issues/27

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/SwedbankPay/Api/Client/ClientVersion.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5523738&bid=18098346)

File: src/SwedbankPay/Api/Client/ClientVersion.php (hunk 1 above)
Comment (payex-dev) [inline src/SwedbankPay/Api/Client/ClientVersion.php:146]: Codacy Issue found: [The use of function file_exists() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5523738)

**Comment by asbjornu** [comment]
The build is failing. Why?

**Comment by aait** [comment]
> The build is failing. Why?

There was 3party problem with GitVersion

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/SwedbankPay/Api/Client/ClientVersion.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5523738&bid=18098346)

File: src/SwedbankPay/Api/Client/ClientVersion.php [hunk 2]
```diff
@@ -139,10 +139,14 @@ private function tryGetVersionNumberFromComposerJson(&$version) : bool
       *
       * @param string $version The by-reference $version variable to assign the version number to, if found.
       * @return bool true if successful; otherwise false.
+      * @SuppressWarnings(PHPMD.CyclomaticComplexity)
       */
     private function tryGetVersionNumberFromComposerLock(&$version) : bool
     {
         $composerLockPath = getcwd() . DIRECTORY_SEPARATOR . 'composer.lock';
+        if (!file_exists($composerLockPath)) {
```
Comment (payex-dev) [inline src/SwedbankPay/Api/Client/ClientVersion.php:147]: Codacy Issue found: [The use of function file_exists() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5523738)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/ClientVersionTest.php  2
         

Complexity decreasing per file
==============================
+ src/SwedbankPay/Api/Client/ClientVersion.php  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5523738&bid=18098346)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/ClientVersionTest.php  2
         

Complexity decreasing per file
==============================
+ src/SwedbankPay/Api/Client/ClientVersion.php  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5523738&bid=18098346)

**Pull request merged**